src/status_im/i18n/i18n_resources.cljs

```diff
@@ -5,8 +5,11 @@
 ;;TODO (14/11/22 flexsurfer) this namespace has been moved to the status-im2 namespace, we keep this only for old (status 1.0) code,
 ;; can be removed with old code later
 
+(def get-keyword
+  (if js/global.__TEST__ str keyword))
```

Comment:
could you please elaborate on why this is needed? how its used

Reply:
This is surprisingly weird given that there are so many other functions in the `cljs.core` namespace, so why would only `cljs.core/keyword` not be defined? At least I hope we don't have to do this kind of change in other namespaces in the future.

Reply:
Let's just hope we don't have to infest other parts of the codebase with these workarounds. I'm super curious to understand why the `keyword` function is not defined, but the other `cljs.core` functions in this namespace are all properly defined.

Reply:
yeah I could not figure that out either. I agree though we don't want to have a lot of parts of the codebase with code like this.